web/app/services/session.ts

```diff
@@ -55,16 +66,18 @@ export default class SessionService extends EmberSimpleAuthSessionService {
   pollForExpiredAuth = keepLatestTask(async () => {
     await simpleTimeout(Ember.testing ? 100 : 10000);
 
-    this.fetch.fetch(
-      "/api/v1/me",
-      {
-        method: "HEAD",
-      },
-      true
-    );
+    // If the reauth message is shown, do nothing but restart the counter.
```

Comment:
I'm thinking of a situation where a user has multiple Hermes tabs open and the session expires: can/should we continue polling to check to see if the `me` API starts returning a 200 again? This way when the user re-auths in one tab, the flash message will go away in all the other tabs?